integration/class-jetpack.php

```diff
@@ -74,15 +73,14 @@ public static function add_sync_comment_meta( $allow_list ) {
 	 * Add custom comment types to the list of comment types.
 	 *
 	 * @param array $comment_types Default comment types.
-	 * @return array
+	 *
+	 * @return array The comment types with ActivityPub types added.
 	 */
 	public static function add_comment_types( $comment_types ) {
-		// jetpack_sync_whitelisted_comment_types runs on plugins_loaded, before comment types are registered.
-		if ( 'jetpack_sync_whitelisted_comment_types' === current_filter() ) {
-			Comment::register_comment_types();
-		}
+		$comment_types[] = 'repost';
+		$comment_types[] = 'like';
```

Comment:
```suggestion
		$comment_types[] = 'like';
		$comment_types[] = 'quote';
```
Assuming we'll ship quotes in 7.6.0?

Reply:
Do we?

Should be in the other PR then ;)